src/Trips.js

```diff
@@ -8,6 +9,16 @@ class Trips {
         this.duration = data.duration
         this.status = data.status
         this.suggestedActivites = data.suggestedActivites
+        this.destination = destination
+        this.cost;
     }
+    
+    //method to return specific user id push to relevant array 
+    //new Date() W3 /MDN 
+    //method calculate the cost of the trip 
+    //method to add agent fee
+    //method to retrieve trips from a certain frame
+    //method to POST new trip to database and add to pending trips
+    //Trips.js, Travelers.js, Desitnations.js, DataRepository.js
 }
-export default Trips
\ No newline at end of file
+export default Trip
```

Comment:
Keep an eye on the "s" for future code! I also keep getting tripped up on these!